cypress/support/commands.js

```diff
@@ -70,32 +70,7 @@ Cypress.Commands.add(
   },
 );
 
-/**
- * Runs necessary test setup to run a clean test.
- */
 Cypress.Commands.add('testSetup', () => {
-  const MAX_RETRIES = 3;
-
-  function retryClearCookies(retryCount) {
```

Comment:
There was a need for this manual script but the newer Cypress version should handle this better